.stylelintrc.json

```diff
@@ -1,6 +1,482 @@
 {
   "extends": "stylelint-config-sass-guidelines",
+  "plugins": [
+    "stylelint-declaration-strict-value"
+  ],
   "rules": {
+    "order/properties-alphabetical-order": null,
+    "order/properties-order": [
```

Comment:
This grouping order is based on [Concentric-CSS](https://github.com/brandon-rhodes/Concentric-CSS/blob/master/style3.css), that uses an outside-inside approach for the rules. Taking from the his source code:
```css
#Concentric-CSS-Overview {
  display: ;    /* Directions about where and how the box is placed */
  position: ;
  float: ;
  clear: ;

  visibility: ; /* Next: can the box be seen? */
  opacity: ;
  z-index: ;

  margin: ;     /* Layers of the box model, from outside to inside */
  outline: ;
  border: ;
  background: ; /* (padding and content BOTH get the background color) */
  padding: ;

  width: ;      /* Content dimensions and scrollbars */
  height: ;
  overflow: ;

  color: ;      /* Textual content */
  text: ;
  font: ;
}
```

Reply:
What about animation/transform/transform? where we can add these properties using groups?

Reply:
In [this link](https://github.com/brandon-rhodes/Concentric-CSS/blob/master/style3.css) we have the complete list of rules which I used to create the order for stylelint.